src/metabase/query_processor/middleware/metrics.clj

```diff
@@ -254,5 +255,6 @@
       (update query :stages #(adjust-metric-stages query nil %))
       (when-let [metric (lib.util.match/match-one <>
                           [:metric _ _] &match)]
+        (prometheus/inc! :metabase-metrics/adjust-errors)
```

Comment:
I guess the `match-one` means we're only counting the # calls that result in at least one failure, not the total number of `:metrics` we failed to adjust.

Reply:
Just noticed there are several places in this ns that throw an exception for incompatible or malformed metrics. Maybe we should count those as well? Hmm, should the metric name be generalized to something like `:metabase-metrics/errors` or should a separate counter be created for each location that throws? 🤔